models/ape_store/base/ape_store_base_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(
+    alias = 'trades',
+    schema = 'ape_store_base',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
```

Comment:
please add incremental predicate property in the model config block. this should filter the target on the same column which the source is being filtered incrementally.

Reply:
wdym with this? i don't understand it 😆

Reply:
it looks like you already have it applied in line 8
`incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],`
maybe you just added since?

to hopefully be more clear:
this property filters the target to the same timeframe as the source, i.e. where you use incremental filter on line 67 for example. based on your model configs, this model will build incrementally via merge statements. this will ensure the merge statement filters the target to a smaller timeframe to scan less rows, therefore becoming more performant